components/gpio_control/example_configs/gpio_settings.ini

```diff
@@ -1,16 +1,18 @@
 [DEFAULT]
 enabled: True
+antibouncehack: False
+
 [VolumeControl]
 enabled: True
 Type: TwoButtonControl ;or RotaryEncoder
-PinUp: 5
-PinDown: 6
-pull_up: True
+Pin1: 5
```

Comment:
How does renaming these parameters work with existing installations, i.e. ini-files which use the old parameter names?

Reply:
Hi @s-martin s-martin. See PR #1406 for my response.